cmd/utils/flags.go

```diff
@@ -500,6 +500,11 @@ var (
 		Usage: "Wait time the rw timer waits for message writing",
 		Value: 15 * time.Second,
 	}
+	MaxRequestContentLengthFlag = cli.IntFlag{
+		Name:  "maxRequestContentLength",
+		Usage: "Max request content length for http, websocket and gRPC",
```

Comment:
```suggestion
		Usage: "Max request content byte length for http, websocket and gRPC",
```

Reply:
I think "content length" is fine. It will not read content as byte, but string. Indeed, "Content-length" is used as an HTTP header name.

Reply:
I changed to `content length in byte`. What do you think? :)